man/labgrid-device-config.1

```diff
@@ -67,11 +67,27 @@ Defaults to \(aqrealm1\(aq.
 .SH IMAGES
 .sp
 The \fBimages:\fP top key provides paths to access preconfigured images to flash
-onto the board.
+onto the board. The image paths can be either relative to the YAML file or
+absolute.
 .SS IMAGE KEYS
 .sp
 The subkeys consist of image names as keys and their paths as values. The
 corresponding name can than be used with the appropriate tool found under TOOLS.
+.SS IMAGE EXAMPLE
+.sp
+Two configured images, one for the root filesystem, one for the bootloader:
+.INDENT 0.0
+.INDENT 3.5
+.sp
+.nf
+.ft C
+images:
+  root: platform\-v7a/root.img
```

Comment:
I'd suggest quoting the string, as this can avoid confusing errors due to yaml type inference.